Kudu.Contracts/Functions/MasterKeyOperation.cs

```diff
@@ -0,0 +1,46 @@
+﻿using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using System;
+
+namespace Kudu.Core.Functions
+{
+    public class MasterKeyOperation : IKeyOperation
+    {
+        public JObject GenerateKeyJson(string masterKey)
+        {
+            JObject hostJson = JObject.Parse($"{{\"masterKey\":{{\"name\":\"master\",\"value\":\"{masterKey}\",\"encrypted\": true }}}}");
```

Comment:
When generating the host secrets JSON, let's make sure the `functionKeys` array is present here, even if empty. We may also want to check the previous behavior, but today, when the host generates the host level secret file, it creates a function secret as well, so we should probably do the same here.